docs/powershell/create-website/index.md

```diff
@@ -22,7 +22,9 @@ This section will detail the functions and which version of IIS the function is
 ### IIS7 Only
 
 * AddAuthoringRule([string] $websiteName, [string] $userName, [string] $access)
-* CheckIfSslBindingExists([string]$webSite, [string]$hostHeader)
+* AddSslCertificate([string] $websiteName, [string] $friendlyName, [string] $hostHeader, [string] $ipAddress="*")
```

Comment:
Can you put `AddSslCertificate([string] $websiteName, [string] $friendlyName, [string] $hostHeader)` and `CreateWebSite([string]$name, [string]$localPath, [string] $appPoolName, [string] $applicationName, [string] $hostName, [string] $logLocation, [int32] $port=80)` from the IIS6 & IIS7 sections into the IIS6 Only section as this then better reflects what users can expect.

Reply:
Yep, makes sense, updated